### PR TITLE
chore: added checkboxes in issue templates

### LINKED ISSUE #51: chore: add a checkbox in all 3 Issue templates
### Description

There are 3 issue templates. add this checkbox shown below in the template's textarea
- [ ] I have read and I am following [the guidelines for creating a new issue](https://github.com/WalkingPal/walking-pal-web/blob/dev/.docs/CONTRIBUTING.md#make-sure-to-follow-these-when-opening-issues)

### PR DESCRIPTION
**Pull request by goverdhan07**
closes #51

### CODE REVIEW

**Comment by ashuvssut** [comment]
I would like to check how does the templates look now in your forked repository 

You can do any one of the two things below.
1. you can merge your [chore/checkbox](https://github.com/goverdhan07/walking-pal-web/tree/chore/checkbox) branch with your [dev](https://github.com/goverdhan07/walking-pal-web/tree/dev) branch.
2. you can temporarily make chore/checkbox branch as `default` branch

I will check the Issue templates after that.

**Comment by goverdhan07** [comment]
> 

Hi @ashuvssut I have fixed the templates, please have a look - [here](https://github.com/goverdhan07/walking-pal-web/issues/new/choose)

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -8,8 +8,15 @@ body:
...
-
+        Write a brief description of the question or issue, also include what you tried and what didn't work. *Please add screenshots if applicable along with* ***steps to reproduce the issue and reference links (if applicable)***. If you need help or if you want to to discuss something with us, join the WalkingPal [Discord community](https://discord.gg/Et3aQTsW6a)!       
         You can assign this issue to yourself if you want to work on it.
     validations:
       required: true
+
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
```
Comment (ashuvssut) [inline .github/ISSUE_TEMPLATE/bug.yml:19]: Remove **all** code of conduct labels

no such labels needed for the checkbox. Rest LGTM

**Review by ashuvssut — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (ashuvssut) [reply .github/ISSUE_TEMPLATE/bug.yml:19]: Also there is a `.idea` folder in you PR. Please Remove that folder

**Comment by ashuvssut** [comment]
Thanks for the PR!

**Review by ashuvssut — APPROVED** [review]

**Pull request merged**